### PR TITLE
[12.0][ADD] project_purchase_link

### PR DESCRIPTION
**Pull request by amcor**
proposal module of @avanzosc

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Modules mustn't have plurals in their names nor generic names that don't define their specific usage, so this should be maybe decoupled for each of the "utilities".

**Comment by oihane** [comment]
I suggest the name of project_purchase_usability. As they are some improvement in usability.

File: project_purchase_utilities/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    "name": "Project Purchase Utilities",
+    "version": "12.0.1.1.0",
+    "license": "AGPL-3",
+    "depends": [
+        "project",
+        "purchase",
+        "hr_timesheet",
+    ],
+    "author": "AvanzOSC",
```
Comment (oihane) [inline project_purchase_utilities/__manifest__.py]: @amcor you forgot to add OCA as author (I did the same mistake) :stuck_out_tongue:

Comment (amcor) [reply project_purchase_utilities/__manifest__.py]: done!

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Comment by amcor** [comment]
@NachoAlesLopez

File: project_purchase_utilities/__manifest__.py [hunk 2]
```diff
@@ -3,11 +3,12 @@
 
 {
     "name": "Project Purchase Utilities",
-    "version": "11.0.1.1.0",
+    "version": "12.0.1.1.0",
```
Comment (nikul-serpentcs) [inline project_purchase_utilities/__manifest__.py]: @amcor should be `12.0.1.0.0`

File: project_purchase_utilities/__manifest__.py [hunk 3]
```diff
@@ -3,11 +3,12 @@
...
     "license": "AGPL-3",
     "depends": [
         "project",
         "purchase",
+        "hr_timesheet",
     ],
     "author": "AvanzOSC",
     "website": "http://www.avanzosc.es",
```
Comment (nikul-serpentcs) [inline project_purchase_utilities/__manifest__.py]: URL should be `https://github.com/OCA/project`

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Review by NachoAlesLopez — APPROVED** [review]
Functional review LGTM.

**Pull request closed without merging**